service_status/service_status.py

```diff
@@ -24,10 +24,10 @@ def _make_grpc_call(self, url, secure=True):
 
             stub = heartb_pb2_grpc.HealthStub(channel)
             try:
-                response = stub.Check(heartb_pb2.HealthCheckRequest(service=""))
+                response = stub.Check(heartb_pb2.HealthCheckRequest(service=""), timeout=SRVC_STATUS_GRPC_TIMEOUT)
             except grpc.RpcError as err:
-                err_code = str(err.code())
-                print(err_code)
+                print(str(err.code()))
```

Comment:
Why have a try block within a try block? Lets just have one try - catch with any exception meaning that the service is down